### ISSUE TITLE
Define and document the technologies used

### ISSUE THREAD

**Issue by maevanapcontact** [maevanapcontact/Ulight-Website#10]
We need to define clearly which technologies, frameworks, tools, and languages we will use in this project.
It should be great to have a file which group everything and add links to the tutorials

**Comment by RA9** [comment]
yeah sure I can help on this one , just need you to suggest some of the technologies to me

**Comment by jennifersmithuk** [comment]
@maevanapcontact one issue on the project. Would you like me to start a doc?
